softwareconfiguration/autofe-reldata/src/main/java/autofe/db/sql/RetrieveInstancesFromDatabaseFailedException.java

```diff
@@ -0,0 +1,14 @@
+package autofe.db.sql;
+
+public class RetrieveInstancesFromDatabaseFailedException extends RuntimeException {

```

Comment:
Make this a checked exception?

Reply:
Resolved, now it is a checked exception.